src/else-dimension-measures.jsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ApplyPreMask } from './masking';
+import { addSeparators } from './utilities';
+
+class ElseDimensionMeasures extends React.PureComponent {
+  render () {
+    const {
+      vFontFamily,
+      vSeparatorCols,
+      measure_count,
+      sufixCells,
+      vSymbolForNulls,
+      vLetterSize,
+      vColorMetric1,
+      vColorMetric1Text,
+      vColorMetric2,
+      vColorMetric2Text,
+      vColorMetric3,
+      vColorMetric3Text,
+      vAllSemaphores,
+      ConceptMatrixPivot,
+      ConceptsAffectedMatrix,
+      vAllMetrics,
+      MetricsAffectedMatrix,
+      vCritic,
+      vMMedium,
+      vNumMeasures,
+      vNumMeasures2,
+      MeasuresFormat,
+      rowNumber,
+      columnText,
+      styleBuilder
+    } = this.props;
+
+    // modified in here
+    let columnNumber,
+      vMaskNum,
+      vColorSemaphore,
+      vColorSemaphoreText,
+      vDivide;
+
+    const measurementCells = [];
+
+    var nMeasure7 = 0;
+    var nMeasure72 = -1;
+    var nMeasure72Semaphore = 0;
+
+    for (var nMeasures22 = 1; nMeasures22 <= vNumMeasures2; nMeasures22++) {
+      nMeasure7++;
+      nMeasure72++;
+      if (columnText.substring(0, 1) === '%') {
+        columnNumber = ApplyPreMask('0,00%', ConceptMatrixPivot[rowNumber][nMeasures22]);
+        var vSpecialF = '0,00%';
+      } else {
+        switch (MeasuresFormat[nMeasure72].substr(MeasuresFormat[nMeasure72].length - 1)) {
+          case 'k':
+            vDivide = 1000;
+            break;
+
+          case 'K':
+            vDivide = 1000;
+            break;
+
+          case 'm':
+            vDivide = 1000000;
+            break;
+
+          case 'M':
+            vDivide = 1000000;
+            break;
+
+          default:
+            vDivide = 1;
+            break;
+        }
+        var vSpecialF = MeasuresFormat[nMeasure72].replace(/k|K|m|M/gi, '');
```

Comment:
isn't vSpecialF already defined.
why don't you define it outside if scope and then change its value at line77 like you do